### DISCUSSION TITLE
3D model black in EDGE and fine in CHROME

### DISCUSSION THREAD

**Discussion by themindfactory** [google/model-viewer#4026] [Q&A]
### Description

Using the same link to the same website, the shark appears black in Edge and OK in Chrome
https://web.dev/model-viewer/

I am having the same issues with my website also, its just easier to show the problem with the demo above.

#### Live Demo
https://web.dev/model-viewer/ open in Edge and Chrome

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [x] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [x] Windows

**Comment by elalish** [comment]
That's not ideal! Since I don't have a windows machine handy, could you paste in any errors/warnings you see in the JS console? Also, do you have the same problem on https://modelviewer.dev/?

**Comment by themindfactory** [comment]
OK this is somewhat odd, I get the error below on the demo site, and also my site... but not on model-viewer site. For the most part no matter what site I go to it mostly works....

model-viewer.js:26890 
THREE.WebGLRenderer: A WebGL context could not be created. Reason:  Could not create a WebGL context, VENDOR = 0x15ad, DEVICE = 0x0405, GL_VENDOR = Google Inc. (VMware), GL_RENDERER = ANGLE (VMware, VMware SVGA 3D Direct3D11 vs_4_0 ps_4_0, D3D11-8.15.1.65), GL_VERSION = 8.15.1.65, Sandboxed = yes, Optimus = no, AMD switchable = no, Reset notification strategy = 0x8252, ErrorMessage = bindToCurrentThread failed: .

**Comment by elalish** [comment]
The site you linked is quite old and still has our legacy fallback version for IE. That's been deprecated for some time; does your site have it as well by chance? If so, try removing it. Just a shot in the dark...

**Comment by themindfactory** [comment]
On my site, all I add is <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>

**Comment by elalish** [comment]
Would you mind trying to build a minimal repro by remixing the glitch link in our issue template? That would go a long way toward helping us fix this, since you have the repro.
